integration_tests/setup/testConstants.ts

```diff
@@ -1,4 +1,4 @@
-const dynamoDbTable = "di-btm-StorageTable-WHKGC5YIMCZF";
+const dynamoDbTable = "di-btm-StorageTable-D6OI93N75URV";
```

Comment:
You could use the `ListTablesCommand` from the dynamodb client to get all the tables then filter so this doesn't need to be updated each run.

Reply:
added ListTables function to get the table name

Reply:
👍